2-mandatory/2-financial-times.js

```diff
@@ -13,25 +20,72 @@ function potentialHeadlines(allArticleTitles) {
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
+//        ["The", "three", "questions", "that", "dominate", "investment"]
+//         fewestNumberOfWords = articleSplitter.length;
+
```

Comment:
This is a very good attempt 👍  One small improvement:
- Could you try doing this without using `let fewestNumberOfWords = 1000;`
- You could either leave it uninitialised and check for `undefined` in your loop.. Or you could maybe initialise to the numberOfWords of the first title in the list.